documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -29,27 +29,9 @@ The display has the following key features:
 * Outer dimensions: 192.96 × 110.76mm
 * Viewable area: 154.08 × 85.92mm
 
-[NOTE]
-====
-If you use Raspberry Pi OS Bullseye or earlier, you can install an on-screen keyboard by typing `sudo apt install matchbox-keyboard` in a terminal. You can also enable right-click emulation by adding the following section to the `/etc/X11/xorg.conf` file.
-
-[source]
-----
-Section "InputClass"
-   Identifier "calibration"
-   Driver "evdev"
-   MatchProduct "FT5406 memory based driver"
-
-   Option "EmulateThirdButton" "1"
-   Option "EmulateThirdButtonTimeout" "750"
-   Option "EmulateThirdButtonMoveThreshold" "30"
-EndSection
-----
-
-These features are not available when running Raspberry Pi OS Bookworm.
-====
+WARNING: The Raspberry Pi 5 does not support automatic display configuration. To attach a display to the `CAM/DISP 1` connector, add `dtoverlay=vc4-kms-dsi-7inch` to `/boot/firmware/config.txt`. To attach a display to the `CAM/DISP 0` connector, add `dtoverlay=vc4-kms-dsi-7inch,dsi0`. Reboot with `sudo reboot` to load the new configuration.
```

Comment:
I believe this **is** now working automatically, with no need for config.txt edits.

Reply:
```suggestion
```
Excellent, thanks! Got caught up in a merge conflict and I didn't want to chance losing this if it was important. Turns out we removed it for a reason.